### DISCUSSION TITLE
Removed Teleport Application Still Redirecting

### DISCUSSION THREAD

**Discussion by jaschadub** [gravitational/teleport#8926] [Q&A] [answered]
Had tested out Teleport with the application setup for Jenkins. Removed all of Teleport but still getting redirected to it for local-IP:8080 (Jenkins)?

Need better uninstall information on site docs.

**Comment by webvictim** [comment, marked as answer]
Could you explain a little more about what you're doing to end up in this situation?

Uninstalling Teleport should just be a case of: 
- terminating any running Teleport processes
- remove `/usr/local/bin/teleport`, `/usr/local/bin/tctl` and `/usr/local/bin/tsh` if you installed manually (or use a package manager to uninstall, if you installed Teleport that way)
- remove `/var/lib/teleport` and `/etc/teleport.yaml`
